README.md

```diff
@@ -139,19 +141,131 @@ fuel-core run \
 --service-name fuel-ignition-node \
 --keypair {KEYGEN_SECRET_KEY} \
 --relayer {ETHEREUM_RPC_ENDPOINT} \
---ip=0.0.0.0 --port 4000 --peering-port 30333 \
+--ip=0.0.0.0 \
+--port 4000 \
+--peering-port 30333 \
+--db-path ~/.fuel-ignition \
+--snapshot {PATH_TO_CHAIN_CONFIGURATION_FOLDER}/ignition \
+--utxo-validation \
+--poa-instant false \
+--enable-p2p \
+--bootstrap-nodes /dnsaddr/mainnet.fuel.network \
+--sync-header-batch-size 100 \
+--relayer-v2-listening-contracts 0xAEB0c00D0125A8a788956ade4f4F12Ead9f65DDf \
+--relayer-da-deploy-height 20620434 \
+--relayer-log-page-size 2000 \
+--sync-block-stream-buffer-size 30 \
+--min-gas-price 1000 \
+--starting-gas-price 1000 \
+--gas-price-change-percent 10 \
+--gas-price-threshold-percent 50
+```
+All the values used for the L1 relayer are based on Ethereum Mainnet make sure `{ETHEREUM_RPC_ENDPOINT}` is a RPC communicating to Ethereum Mainnet network.
+
+Instead of directly placing your personal values on the command we advise you to use, for example, environment variables.
+
+### From source
+
+Clone the chain configuration folder :
+```
+git clone https://github.com/FuelLabs/chain-configuration
+```
+
+Clone the `fuel-core` repository : 
+```
+git clone https://github.com/FuelLabs/fuel-core.git
+```
+
+Go to the latest release tag for ignition on the `fuel-core` repository :
+```
+git checkout v0.40.2
+```
+
+Generate a keypair for your node :
+```
+cargo run --bin fuel-core-keygen -- new --key-type peering
+```
+and copy the secret key displayed.
+
+Run your node (change all variable with {} to your own personal variables):
+```bash
+cargo run --release --bin fuel-core -- run \
+--enable-relayer \
+--service-name fuel-ignition-node \
+--keypair {KEYGEN_SECRET_KEY} \
+--relayer {ETHEREUM_RPC_ENDPOINT} \
+--ip=0.0.0.0 \
+--port 4000 \
+--peering-port 30333 \
 --db-path ~/.fuel-ignition \
 --snapshot {PATH_TO_CHAIN_CONFIGURATION_FOLDER}/ignition \
---utxo-validation --poa-instant false --enable-p2p \
+--utxo-validation \
+--poa-instant false \
+--enable-p2p \
 --bootstrap-nodes /dnsaddr/mainnet.fuel.network \
 --sync-header-batch-size 100 \
---relayer-v2-listening-contracts=0xAEB0c00D0125A8a788956ade4f4F12Ead9f65DDf \
---relayer-da-deploy-height=20620434 \
---relayer-log-page-size=100 \
---sync-block-stream-buffer-size 30
+--relayer-v2-listening-contracts 0xAEB0c00D0125A8a788956ade4f4F12Ead9f65DDf \
+--relayer-da-deploy-height 20620434 \
+--relayer-log-page-size 2000 \
+--sync-block-stream-buffer-size 30 \
+--min-gas-price 1000 \
+--starting-gas-price 1000 \
+--gas-price-change-percent 10 \
+--gas-price-threshold-percent 50
```

Comment:
We don't need to provide information which fields to use, for that we have documentation and we can point to it. Maybe we need to rework "Running a Ignition node" and point to the documentation to avoid multiple places.

We just need to describe how to build the production ready binary.

Reply:
I don't see where is the documentation. Can you point it out to me ? I think it's valuable to be able to copy-paste a command that is working as expected to connect to Ignition/Launch a local network and then if you need something specific you can just tweak some parameters. We saw in the past weeks that if some parameters are not present the node isn't working as expected so to avoid confusion I find it useful to first set everything correctly (such as a `--ignition` that we could have) and then play with flags if you want.

Reply:
https://docs.fuel.network/guides/running-a-node/running-a-mainnet-node/